release-notes/opendistro-for-elasticsearch-release-notes-1.11.0.md

```diff
@@ -0,0 +1,315 @@
+# Open Distro for Elasticsearch 1.11.0 Release Notes
+
+## Release Highlights
+
+## Release Details
+
+You can also track upcoming features in Open Distro for Elasticsearch by watching the code repositories or checking the project website (https://opendistro.github.io/for-elasticsearch/features/comingsoon.html).
```

Comment:
Square brackets around project website [project website]  (https://opendistro.github.io/for-elasticsearch/features/comingsoon.html)

Reply:
Updated